ci/vm/provision1.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -Exeuo pipefail
+
+OS=$1
+KERNEL=$2
+
+install_packages() {
+  if [[ $OS == *"ubuntu"* ]]; then
+    apt-get update &&
+      DEBIAN_FRONTEND=noninteractive apt-get install -yq \
+        automake \
+        dkms \
+        git \
+        libtool &&
+      apt-get clean && rm -rf /var/lib/apt/lists/*
+  elif [[ $OS == *"centos"* ]]; then
+    yum install -y -q centos-release-scl
+    yum install -y -q \
+      automake \
+      devtoolset-8-gcc \
+      devtoolset-8-gcc-c++ \
+      elfutils-libelf-devel \
+      git \
+      libtool \
+      make &&
+      yum clean all
+  fi
+}
+
+install_mainline_kernel() {
+    uname -r
+    add-apt-repository ppa:cappelikan/ppa    
```

Comment:
Where is this repo from? Is it something official?

Reply:
The 'mainline' utility requires this repo
https://github.com/bkw777/mainline